compact/hcal/forward_insert.xml

```diff
@@ -139,11 +139,16 @@
   </detectors>
 
   <readouts>
+    <documentation>
+      Readout is segmented into hexagons with sides of 1.889 cm
+      The hexagons positions cycle through four sets of layer configurations ("H4 staggering")
+      i.e. hexagons in layers 1-4 (5-8, ...) are all offset transversely from each other
+    </documentation>
     <readout name="HcalEndcapPInsertHits">
       <segmentation
-        type="CartesianGridXY"
-        grid_size_x="3.*cm"
-        grid_size_y="3.*cm"
+        type="HexGrid"
+        side_length="1.889*cm"
+        stagger="2"
```

Comment:
Does this need a unit? Or is this a type of staggering?

Reply:
It's a type of staggering. The `stagger="2"` reflects the staggering described in lines 144/145.

Reply:
Alright, added more description and some links to the segmentation code.